src/app/dts/dts.service.ts

```diff
@@ -25,7 +25,9 @@ export class DtsService {
    * Retrieves and indication is the backend DTS service is healthy.
    */
   public getStatus(): Observable<string> {
-    return this.http.get(this.url + 'status', {responseType: 'text'});
+    return this.http.get(this.url + 'status', {responseType: 'text'}).pipe(catchError( error => {
+      return throwError(error);
```

Comment:
This line could be simplified into:
`.pipe(catchError(throwError))`

Reply:
I believe the recommendation won't work as throwError is a function which requires the "error" as a parameter.

Reply:
Well, passing a reference to a function as a function does work in Javascript :-) Give it a try, you'll see.